Tests/Onboarding/IntroViewControllerTests.swift

```diff
@@ -0,0 +1,55 @@
+import GDSAnalytics
+import GDSCommon
+@testable import OneLogin
+import XCTest
+
+final class IntroViewControllerTests: XCTestCase {
+    var loginSession: MockLoginSession!
+    var mockAnalyticsService: MockAnalyticsService!
+    var sut: IntroViewController!
+    
+    override func setUp() {
+        super.setUp()
+        
+        loginSession = MockLoginSession(window: UIWindow())
+        mockAnalyticsService = MockAnalyticsService()
+        sut = ViewControllerFactory(analyticsService: mockAnalyticsService).createIntroViewController(session: loginSession)
+    }
+    
+    override func tearDown() {
+        loginSession = nil
+        sut = nil
+        
+        super.tearDown()
+    }
+}
+
+extension IntroViewControllerTests {
```

Comment:
Establish MockLoginSessionConfiguration to assert for values in session configuration

Reply:
@elishkaflint
A conversation with @Oliver-Binns about the value of doing has led me to believe that an interface for the session configuration is unneeded for now. The configuration should be flexible enough to input things in, we can test the instance we have the project code, I've just included that test in my last commit.
Do let me know if you think there's anything I'm missing in my understanding or you'd like a further conversation about it 🙂